### PR TITLE
Adds sound to Smoke Grenades

### PR DESCRIPTION
**Pull request by Hmeister-fake**
## About the PR
Adds sound to both smoke grenades and teargas grenades

## Why / Balance
They lacked it beforehand, now they shouldnt.

## Technical details
I am not a coder, this is just a hacky copy and paste job of @deltanedas code from [#22855] with some assumptions being made, but the sound is original. First ever PR, this will hopefully work as intended.

## Media
- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

- tweak: Smoke grenades now have sound.

### CODE REVIEW

**Comment by deltanedas** [comment]
add to attributions.yml in effects too

File: Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml [hunk 1]
```diff
@@ -360,6 +360,9 @@
   - type: SoundOnTrigger
     sound: /Audio/Effects/smoke.ogg
   - type: DeleteOnTrigger
+    - type: TimerTriggerVisuals
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml:363]: unindent

File: Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml [hunk 2]
```diff
@@ -376,3 +379,6 @@
       reagents:
       - ReagentId: TearGas
         Quantity: 50
+            - type: TimerTriggerVisuals
+    primingSound:
+      path: /Audio/Effects/Smoke-grenade.ogg
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml:384]: delete

**Review by deltanedas — CHANGES_REQUESTED** [review]
sound is paid access and has no easily visible license ultra sus

**Comment by Hmeister-fake** [comment]
> sound is paid access and has no easily visible license ultra sus

i just downloaded it and then used it with a custom sound, so it should be fine.

**Comment by metalgearsloth** [comment]
> > sound is paid access and has no easily visible license ultra sus
> 
> i just downloaded it and then used it with a custom sound, so it should be fine.

The project hasn't paid for the sound though?

**Comment by deathride58** [comment]
image

The sound's license ((seen here)[https://www.pond5.com/legal/license]) is fairly clear that this use is a *hard* no, considering there's no way to have a sound in the repo without allowing raw access in the process.

**Comment by Emisse** [comment]
change the sound effect to something we can use

**Comment by Hmeister-fake** [comment]
New sound is now completely copyright free, all good.

**Comment by EmoGarbage404** [comment]
@Hmeister-fake the sound effect you used is not licensed to be used in this project due to the stipulation on hosting content on other websites.

image

Additionally, the copyright you supplied in the attribution is incorrect. In the future, make sure you verify the actual license for audio being used and that it is suitable for the project.

Thank you.

**Comment by Hmeister-fake** [comment]
> Additionally, the copyright you supplied in the attribution is incorrect. In the future, make sure you verify the actual license for audio being used and that it is suitable for the project.

It shouldn't be? The actual smoke effect is taken from an unlicensed sound I already had that doesn't actually have a licence I know of (I don't remember what it's from but not a website) but the grenade click is however from a free licence thing. Should I change the effect to be pre-existing from ss13, should I create the noise myself or do you have any suggestions on what I should do.

**Comment by deltanedas** [comment]
not having a license doesnt mean CC0-1.0 it means all rights reserved

**Pull request merged**